app/helpers/sufia_helper.rb

```diff
@@ -15,4 +15,18 @@ def link_to_fields(fields, field_string)
     fields.map { |tag| link_to_facet(tag, field_string) }
   end
 
+  def link_to_field(fieldname, fieldvalue, displayvalue = nil)
```

Comment:
Can you add a comment about why this needs to be here and what it does differently? Looks like it's only used for nested_authors?